<ui version="4.0" >
 <class>MiniWebBrowser</class>
 <widget class="QWidget" name="MiniWebBrowser" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Mini Web Browser</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QToolButton" name="toolButtonBack" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Go back</string>
       </property>
       <property name="statusTip" >
        <string>Go back</string>
       </property>
       <property name="icon" >
        <iconset resource="qmc2.qrc" >
         <normaloff>:/data/img/back.png</normaloff>:/data/img/back.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonForward" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Go forward</string>
       </property>
       <property name="statusTip" >
        <string>Go forward</string>
       </property>
       <property name="icon" >
        <iconset resource="qmc2.qrc" >
         <normaloff>:/data/img/forward.png</normaloff>:/data/img/forward.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonReload" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Reload current URL</string>
       </property>
       <property name="statusTip" >
        <string>Reload current URL</string>
       </property>
       <property name="icon" >
        <iconset resource="qmc2.qrc" >
         <normaloff>:/data/img/reload.png</normaloff>:/data/img/reload.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonStop" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Stop loading of current URL</string>
       </property>
       <property name="statusTip" >
        <string>Stop loading of current URL</string>
       </property>
       <property name="icon" >
        <iconset resource="qmc2.qrc" >
         <normaloff>:/data/img/stop_browser.png</normaloff>:/data/img/stop_browser.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonHome" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Go home (first page)</string>
       </property>
       <property name="statusTip" >
        <string>Go home (first page)</string>
       </property>
       <property name="icon" >
        <iconset resource="qmc2.qrc" >
         <normaloff>:/data/img/home.png</normaloff>:/data/img/home.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditURL" >
       <property name="toolTip" >
        <string>Current URL</string>
       </property>
       <property name="statusTip" >
        <string>Current URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonLoad" >
       <property name="toolTip" >
        <string>Load URL</string>
       </property>
       <property name="statusTip" >
        <string>Load URL</string>
       </property>
       <property name="icon" >
        <iconset resource="qmc2.qrc" >
         <normaloff>:/data/img/yes.png</normaloff>:/data/img/yes.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QWebView" name="webViewBrowser" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>toolButtonBack</tabstop>
  <tabstop>toolButtonForward</tabstop>
  <tabstop>toolButtonReload</tabstop>
  <tabstop>toolButtonStop</tabstop>
  <tabstop>toolButtonHome</tabstop>
  <tabstop>lineEditURL</tabstop>
  <tabstop>toolButtonLoad</tabstop>
  <tabstop>webViewBrowser</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>toolButtonBack</sender>
   <signal>clicked()</signal>
   <receiver>webViewBrowser</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>26</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel" >
     <x>53</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonForward</sender>
   <signal>clicked()</signal>
   <receiver>webViewBrowser</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>58</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel" >
     <x>90</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonReload</sender>
   <signal>clicked()</signal>
   <receiver>webViewBrowser</receiver>
   <slot>reload()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>86</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel" >
     <x>122</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonStop</sender>
   <signal>clicked()</signal>
   <receiver>webViewBrowser</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>122</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel" >
     <x>156</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditURL</sender>
   <signal>returnPressed()</signal>
   <receiver>toolButtonLoad</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>416</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel" >
     <x>493</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
